### ISSUE TITLE
get_replay only works on GET requests

### ISSUE THREAD

**Issue by dogwong** [ppy/osu-api#200]
Whatever parameters I used and it just simply return `{"error":"Replay not available."}
`
and this costs me few days to find out the reason... (stupid me...)

Because I'm using POST request while other endpoints is just working fine

**Comment by omkelderman** [comment]
imo the real bug here is that all those other endpoints (aparently) even respond to a POST request. Doesnt make any sense to me whatsoever xD

**Comment by dogwong** [comment]
yep... at first I thought it is just my problem, Eg wrong beatmap id or user id, or even because of aged records...

Until I grab the same request and type it in Chrome's address bar and press enter :neutral_face:
